protocol/chainlib/common.go

```diff
@@ -30,6 +30,7 @@ const (
 	relayMsgLogMaxChars        = 200
 	RPCProviderNodeAddressHash = "Lava-Provider-Node-Address-Hash"
 	RPCProviderNodeExtension   = "Lava-Provider-Node-Extension"
+	RpcProviderLoadRateHeader  = "Lava-provider-load-rate"
```

Comment:
Please match case with the rest of the headers:
```suggestion
	RpcProviderLoadRateHeader  = "Lava-Provider-Load-Rate"
```

Reply:
Fixed in [9458d6c](https://github.com/lavanet/lava/pull/1720/commits/9458d6c71c4809897bbaecf7b11742151c064e2a)